.github/workflows/podman-aws-e2e.yaml

```diff
@@ -40,22 +40,24 @@ jobs:
 
       - name: Setup buildkit
         uses: docker/setup-buildx-action@v2
-      
+
       - name: Configure AWS Credentials
         uses: aws-actions/configure-aws-credentials@v4
         with:
           role-to-assume: arn:aws:iam::${{ secrets.AWS_ACCOUNT_ID }}:role/konvoy-image-builder
           aws-region: us-west-2
-      
+
       - name: Download GoReleaser
         run: go install github.com/goreleaser/goreleaser@v1.15.2
-      
+
       - name: Build snapshot
-        run: make build.snapshot
+        run:  |-
+          make build-for-podman
+          goreleaser build --id konvoy-image-wrapper-for-podman
 
       - name: Run E2E test for AWS centos 7.9 using podman
         run: |-
-          KIB_CONTAINER_ENGINE=podman dist/konvoy-image-wrapper_linux_amd64_v1/konvoy-image build aws images/ami/centos-79.yaml --dry-run
+          KIB_CONTAINER_ENGINE=podman dist/konvoy-image-wrapper-for-podman_linux_amd64_v1/konvoy-image build aws images/ami/centos-79.yaml --dry-run
```

Comment:
Why do we need a separate binary?

Reply:
build.snapshot is seeing disk space issues. i created a separate gorelaser target which only builds the amd64 image for podman

Reply:
furthermore, this allows us to execute the test using an embeded binary.